awx/ui/Makefile

```
@@ -87,7 +87,7 @@ ui/src/webpack: $(UI_DIR)/src/node_modules/webpack
 ## True target for ui/src/webpack.
 $(UI_DIR)/src/node_modules/webpack:
 	@echo "=== Installing webpack ==="
-	@cd $(UI_DIR)/src && npm install webpack
+	@cd $(UI_DIR)/src && n 20 && npm install webpack
```

given the official docs still recommend nodejs 18, maybe we default to 18?

```suggestion
	@cd $(UI_DIR)/src && n 18 && npm install webpack
```